test/interpreter_functional/test_suites/run_pipeline/index.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { delay } from 'bluebird';
+
+export default function ({ getService, getPageObjects, loadTestFile }) {
+  const browser = getService('browser');
+  const esArchiver = getService('esArchiver');
+  const kibanaServer = getService('kibanaServer');
+  const appsMenu = getService('appsMenu');
+  const PageObjects = getPageObjects(['common', 'header']);
+
+  describe('runPipeline', function () {
+    before(async () => {
+      await esArchiver.loadIfNeeded('../functional/fixtures/es_archiver/logstash_functional');
+      await esArchiver.load('../functional/fixtures/es_archiver/visualize_embedding');
+      await kibanaServer.uiSettings.replace({ 'dateFormat:tz': 'Australia/North', 'defaultIndex': 'logstash-*' });
+      await browser.setWindowSize(1300, 900);
+      await PageObjects.common.navigateToApp('settings');
+      await appsMenu.clickLink('Run Pipeline');
+      await delay(5000);
```

Comment:
Why is this necessary? Do all pipeline tests need to complete in 5s?  What about waiting for the App to tell us the pipeline tests are complete via a data-test-subj?

Reply:
this is delay from clicking the menu link till the time app actually loads ... 
not sure if its really needed, even if atm it is, we probably have a better way to detect when app has finished loading ?